src/Microsoft.Owin.Security.Authorization/AuthorizationHelper.cs

```diff
@@ -54,12 +54,13 @@ public AuthorizationHelper(IOwinContextAccessor owinContextAccessor)
                 throw new InvalidOperationException("AuthorizationOptions.Dependencies must not be null");
             }
 
-            var authorizationService = GetAuthorizationService(options);
-            var policy = AuthorizationPolicy.Combine(options, new[] {authorizeAttribute});
+            var policyProvider = new DefaultAuthorizationPolicyProvider(options);
```

Comment:
At first glance, this line looks like it ignore's the policy provider in `AuthorizeDependencies`.  After reviewing the rest of the code, I see how it does not.  Still, we should strive to keep things as obvious as possible for other developers.  Perhaps we can get away with doing something like 
```cs
IAuthorizationPolicyProvider provider;
if (options.Dependencies.PolicyProvider != null)
{
    provider = options.Dependencies.PolicyProvider;
}
else 
{
    provider = new DefaultAuthorizationPolicyProvider(options);
}
```
and removing the` IAuthorizationPolicyProvider` implementation from inside `AuthorizationOptions` so that it is more obvious.